fixtures/vitest-pool-workers-examples/workflows/README.md

```diff
@@ -0,0 +1,8 @@
+# 🤯 rpc
+
+This Worker defines `WorkerEntrypoint` default and named exports. It also defines Durable Objects subclassing `DurableObject`. All of these classes define properties and methods that can be called over RPC. Integration tests dispatch events, access properties and call methods using the `SELF` helper from the `cloudflare:test` module.
+
+| Test                                                      | Overview                                     |
+| --------------------------------------------------------- | -------------------------------------------- |
+| [integration-self.test.ts](test/integration-self.test.ts) | Integration test using `SELF`                |
+| [unit.test.ts](test/unit.test.ts)                         | Unit tests calling exported classes directly |
```

Comment:
You might wanna modify this to Workflow? We point people to these examples all the time. So it is best to tell people Workflow support is not ready yet in the readme.